code/modules/clothing/head/helmet.dm

```diff
@@ -218,9 +218,9 @@
 
 /obj/item/clothing/head/helmet/swat
 	name = "\improper SWAT helmet"
-	desc = "These are often used by highly trained SWAT Members."
-	icon_state = "swat"
-	armor_list = list(melee = 80, bullet = 60, energy = 40, bomb = 50, bio = 10, rad = 0)
+	desc = "An oldie but a goodie. This helmet was often the last thing seen by those so foolish as to draw the attention of the Solarian Frontier enforcement corps. Judging by the dents, it didn't save its last wearer, but perhaps you'll fare better?"
```

Comment:
this means EVERY swat helm is dented and used. bit of an odd flavour

Reply:
there are no swat helmets other than the ones i placed for this particular thing.